interface/patient_file/pos_checkout.php

```diff
@@ -157,978 +151,972 @@ function generate_receipt($patient_id, $encounter = 0)
-    }
-
-    echo " <tr>\n";
-    echo "  <td>" . text(oeFormatShortDate($date));
-    echo "<input type='hidden' name='line[$lino][code_type]' value='" . attr($code_type) . "'>";
-    echo "<input type='hidden' name='line[$lino][code]' value='" . attr($code) . "'>";
-    echo "<input type='hidden' name='line[$lino][id]' value='" . attr($id) . "'>";
-    echo "<input type='hidden' name='line[$lino][description]' value='" . attr($description) . "'>";
-    echo "<input type='hidden' name='line[$lino][taxrates]' value='" . attr($taxrates) . "'>";
-    echo "<input type='hidden' name='line[$lino][price]' value='" . attr($price) . "'>";
-    echo "<input type='hidden' name='line[$lino][units]' value='" . attr($units) . "'>";
-    echo "</td>\n";
-    echo "  <td>" . text($description) . "</td>";
-    echo "  <td align='right'>" . text($units) . "</td>";
-    echo "  <td align='right'><input type='text' name='line[$lino][amount]' " .
-       "value='" . attr($amount) . "' size='6' maxlength='8'";
-  // Modifying prices requires the acct/disc permission.
-  // if ($code_type == 'TAX' || ($code_type != 'COPAY' && !acl_check('acct','disc')))
-    echo " style='text-align:right;background-color:transparent' readonly";
-  // else echo " style='text-align:right' onkeyup='computeTotals()'";
-    echo "></td>\n";
-    echo " </tr>\n";
-    ++$lino;
-}
-
-// Create the taxes array.  Key is tax id, value is
-// (description, rate, accumulated total).
-$taxes = array();
-$pres = sqlStatement("SELECT option_id, title, option_value " .
-  "FROM list_options WHERE list_id = 'taxrate' AND activity = 1 ORDER BY seq, title, option_id");
-while ($prow = sqlFetchArray($pres)) {
-    $taxes[$prow['option_id']] = array($prow['title'], $prow['option_value'], 0);
-}
-
-// Print receipt header for facility
-function printFacilityHeader($frow)
-{
-    echo "<p><b>" . text($frow['name']) .
-    "<br>" . text($frow['street']) .
-    "<br>" . text($frow['city']) . ', ' . text($frow['state']) . ' ' . text($frow['postal_code']) .
-    "<br>" . text($frow['phone']) .
-    "<br>&nbsp" .
-    "<br>";
-}
-
-// Pring receipt header for Provider
-function printProviderHeader($pvdrow)
-{
-    echo "<p><b>" . text($pvdrow['title']) . " " . text($pvdrow['fname']) . " " . text($pvdrow['mname']) . " " . text($pvdrow['lname']) . " " .
-    "<br>" . text($pvdrow['street']) .
-    "<br>" . text($pvdrow['city']) . ', ' . text($pvdrow['state']) . ' ' . text($pvdrow['postal_code']) .
-    "<br>" . text($pvdrow['phone']) .
-    "<br>&nbsp" .
-    "<br>";
-}
-
-// Mark the tax rates that are referenced in this invoice.
-function markTaxes($taxrates)
-{
-    global $taxes;
-    $arates = explode(':', $taxrates);
-    if (empty($arates)) {
-        return;
-    }
-
-    foreach ($arates as $value) {
-        if (!empty($taxes[$value])) {
-            $taxes[$value][2] = '1';
+        
+        <?php Header::setupHeader(['bootstrap', 'datetime-picker','font-awesome']);?>
```

Comment:
remove bootstrap and font-awesome (these are included by default)

Reply:
Done